nodejs/json-rpc-relay/integration.tests/mirror.node/web3.clientVersion.test.js

```diff
@@ -0,0 +1,69 @@
+console.clear();
+
+require('dotenv').config({path: '.env'});
+const Web3 = require('web3');
+const { curly } = require('node-libcurl');
+const { ethers } = require("ethers");
+
+const httpHeaderJson = [
+    'Content-Type: application/json',
+    'Accept: application/json'
+  ];
+const arkhiaJsonRpcRelayTestnet  = `${process.env.ARKHIA_TESTNET_URL}/${process.env.ARKHIA_API_KEY}`;
+const communityHashioMainnet = process.env.COMMUNITY_MAINNET_URL;
+
+const getPayload = () => {
+    const data = {
+        "jsonrpc": "2.0",
+        "method": "web3_clientVersion",
+        "params": [],
+        "id": 0
+      };
+    return data;
+}
+
+describe('[CURL] ClientVersion', () => {
+
+    test('Should get relay version from Arhia Testnet', async () => {
+        // Arrange
+        const configPayload = getPayload();
+
+        // Act
+        const { data } = await curly.post(arkhiaJsonRpcRelayTestnet, {
+            postFields: JSON.stringify(configPayload),
+            httpHeader: httpHeaderJson,
+        });
+
+        // Assert
+        expect(data?.result).toBeDefined();
```

Comment:
Should we assert the version ? If the tests fail after the upgrade that's actually ok